### PR TITLE
Misc docs improvements

### PR DESCRIPTION
**Pull request by scarletfog**
### Context
Several smaller things to improve - better wording, styling for tables, removal of the roadmap page (as requested).

### How has this been tested?
I ran the documentation locally.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

### Related issue(s):
1. https://github.com/handsontable/hyperformula/issues/572

### CODE REVIEW

**Comment by scarletfog** [comment]
@wojciechczerniak I need your help here with two things - 
1. I could not resolve an issue we have with "help us improve this page" that leads to 404, for example: https://handsontable.github.io/hyperformula/api/ - I tried to play with frontmatter as I found this issue: https://github.com/vuejs/vuepress/issues/1762 but without success - it's displayed just like a normal text.
2. Which file could be used to count the current number of available functions? In the config file, I saw we have `HyperFormula.version`, `HyperFormula.buildDate`, `HyperFormula.releaseDate` and I suspect we should add something similar: `HyperFormula.functionsNumber` to be later used as: `{{ $page.functionsNumber }}`.

**Comment by wojciechczerniak** [comment]
@scarletfog see d1f41b2c90f754206f3ad99a1c9cc38b1ebc338e

**Comment by scarletfog** [comment]
Ah, thank you so much @wojciechczerniak!

File: docs/guide/built-in-functions.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ because a spreadsheet is probably the most universal software
 ever created. We wanted the same flexibility for HyperFormula
 but without the constraints of the spreadsheet UI.
 
-All implemented functions are grouped into 8 different categories:
+The latest version of HyperFormula has an extensive collection of **{{ $page.functionsCount }}** functions in 9 categories:
```
Comment (scarletfog) [inline docs/guide/built-in-functions.md:13]:
```suggestion
The latest version of HyperFormula has an extensive collection of **{{ $page.functionsCount }}** functions grouped into 9 categories:
```

**Review by wojciechczerniak — APPROVED** [review]
Awesome! 🥇

**Pull request merged**